gadget/mkfs.go

```diff
@@ -34,16 +34,13 @@ import (
 // directory.
 func MkfsExt4(img, label, contentsRootDir string) error {
 	// taken from ubuntu-image
+	// Filesystem creation is used only in snap-bootstrap for uc20 install.
+	// See https://github.com/snapcore/snapd/pull/6997#discussion_r293967140
+	// and https://bugs.launchpad.net/snappy/+bug/1878374
```

Comment:
// Switched to use mkfs defaults for https://bugs.launchpad.net/snappy/+bug/1878374
// For caveats/requirements in case we need support for older systems:
// https://github.com/snapcore/snapd/pull/6997#discussion_r293967140